backend/src/main/resources/apis/api.yaml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.0
+info:
+  version: 0.0.1
+  title: Bonds For Success Api
+  description: Api outlining the rest calls for the Bonds For Success project
+
+servers:
+  - url: localhost:8080
+
+paths:
+  /helloWorld:
+    get:
+      description: Hello World
+      responses:
+        '200':
+          description: OK
+          content:
+            text/plain:
+              schema:
+                type: string
+  /createTable:
+    post:
+      description: GIVEN a mentee and mentor table THEN return a merged table
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                menteeFile:
+                  type: string
+                  format: binary
+                mentorFile:
+                  type: string
+                  format: binary
+      responses:
+        '200':
+          description: Successfully created merged table
+          content:
+            text/plain:
+              schema:
+                type: array
+                items:
+                  type: Match
+
+components:
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+  schemas:
+    Match:
+      type: object
+      properties:
+        mentor:
+          type: Person
+        mentee:
+          type: Person
+        keywordsShared:
+          type: array
+          items:
+            type: string
+          uniqueItems: true
+      required:
+        - mentor
+        - mentee
+
+    Person:

```

Comment:
Does type person not need the emails of each person? Since each person's email is listed in the example excel sheet that was give to us